### ISSUE TITLE
PropType of waveColor to support gradients

### ISSUE THREAD

**Issue by bianchimro** [mspae/react-wavesurfer#56]
Hi,
it would be great if we could use a gradient for waveColor option (as it happens at the top of wavesurfer-js.org page)

doing this
```
var ctx = document.createElement('canvas').getContext('2d');
var linGrad = ctx.createLinearGradient(0, 64, 0, 200);
linGrad.addColorStop(0.5, 'white');
linGrad.addColorStop(0.5, 'crimson');

<Wavesurfer
          options={{barHeight:4, barWidth:2, waveColor:linGrad }}
          audioFile={`someaudio.mp4`}
        />
```

Works for me but a warning is thrown as the "waveColor" property is expected to be a string.

**Comment by mspae** [comment]
I'm open for a PR for this!

**Comment by mspae** [comment]
fixed in v0.8.5
